Samples/auth/Outlook-Add-in-SSO-NAA-Identity/README.md

```diff
@@ -0,0 +1,249 @@
+
+1. Still on the same app registration, select the **App roles** blade to the left.
+1. Select **Create app role**:
+    1. For **Display name**, enter a suitable name for your application permission, for instance `Todolist.Read.All`.
+    1. For **Allowed member types**, choose **Application** to ensure other applications can be granted this permission.
+    1. For **Value**, enter `Todolist.Read.All`.
+    1. For **Description**, enter `Allows the app to read the signed-in user's files.`
+    1. Select **Apply** to save your changes.
+
+Repeat the previous steps to add another app permission named `Todolist.ReadWrite.All`.
+
+### Configure the service app (Contoso-Web-API-Server) to use your app registration
+
+Open the sample project in Visual Studio Code to configure the code. In the steps below, "ClientID" is the same as "Application ID" or "AppId".
+
+1. Open the API/server-helpers/authConfig.js file.
+1. Find the key `Enter_API_Application_Id_Here` and replace the existing value with the application ID (clientId) of Contoso-Web-API-Server app copied from the Microsoft Entra admin center.
+1. Save the file.
+
+### Register the client app (Contoso-Outlook-Add-in)
+
+1. Go to the [Azure portal - App registrations](https://go.microsoft.com/fwlink/?linkid=2083908) page to register your app.
+1. Sign in with the **_admin_** credentials to your Microsoft 365 tenancy. For example, **MyName@contoso.onmicrosoft.com**.
+1. Select **New registration**. On the **Register an application** page, set the values as follows.
+
+   - Set **Name** to `Contoso-Outlook-Add-in`.
+   - Set **Supported account types** to **Accounts in this organizational directory only**.
+   - Select **Register**.
+
+1. In the **Overview** blade, find and note the **Application (client) ID**. You use this value in your app's configuration file(s) later in your code.
+1. In the app's registration screen, select the **Authentication** blade to the left.
+1. If you don't have a platform added, select **Add a platform** and select the **Single-page application** option.
+    1. In the **Redirect URI** section enter the following redirect URIs:
+        1. `http://localhost:3000`
+        1. `http://localhost:3000/redirect`
+        1. `brk-multihub://localhost:3000`
+    1. Click **Save** to save your changes.
+
+Since this app signs-in users, we will now proceed to select **delegated permissions**, which is is required by apps signing-in users.
```

Comment:
```suggestion
Since this app signs-in users, you'll now proceed to select **delegated permissions**, which is required by apps signing-in users.
```